lib/plugins/exchange/ccxt.js

```diff
@@ -0,0 +1,51 @@
+var ccxt = require('ccxt')
+const coinUtils = require('../../coin-utils')
+const common = require('../common/ccxt')

```

Comment:
use desconstruct for common too, just like consts we don't really have reason to hold the entire object

```js
const { ALL_EXCHANGES, buildMarket } = require('../common/ccxt')
```